testing/btest/spicy/tuple-arg.zeek

```diff
@@ -27,7 +27,7 @@ public type Banner = unit {
     dash    : /-/;
     software: /[^\r\n]*/;
 
-    unset_field: uint64 if ( False );
+    unset_field: bytes &eod if ( False );
```

Comment:
Not completely clear to me what this does and whether it is related to the baseline change. The test seems to still fail for me.

Reply:
The issue is in the EVT file, which passes `unset_field` to the `ssh::banner` event, which expected as `bytes`.